randovania/game_connection/connector/am2r_remote_connector.py

```diff
@@ -170,14 +170,15 @@ def new_inventory_received(self, new_inventory_response: str) -> None:
                     "Response should contain a '|', but it doesn't. Original response: %s", new_inventory_response
                 )
                 continue
-            (item_name, quantity) = position.split("|", 1)
-            if not quantity.isdigit():
+            (item_name, q) = position.split("|", 1)
+            if not q.isdigit():
```

Comment:
does python not have something akin to `bool TryParseInt(string, out int)`?

Reply:
Apparently nothing from the standard library. Would require writing a custom function akin to 
```py
def try_parse_int(s: str): -> int | None:
  if s.isdigit():
    return int(s)
  return None
```

Which afaik we don't need right now and also doesn't solve the issue of needing to explicitly cast after